interview-questions.md

```diff
@@ -8,62 +8,62 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-  Your answer:
+  Your answer: Parameters are used in creating and declaring a function. While arguments are used when invoking the function.
 
-  Researched answer:
+  Researched answer: Function parameters specify a single or list of variables when declaring a function. Function arguments are passing/assigning value to a declared function when invoked. 
 
 
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-  Your answer:
+  Your answer: getorDefault uses the key value and general purpose is a static method
 
-  Researched answer:
+  Researched answer: General purpose the key and value  components that have to be put in. getorDefault is the output value to the corresponding key and if there is none it can simply be returned so the general purpose is required 
```

Comment:
The current element or value is the required parameter in map, whereas index and array are optional.